src/BloomBrowserUI/bookEdit/OverflowChecker/OverflowChecker.js

```diff
@@ -1,62 +1,48 @@
-/// <reference path="../../lib/jquery.d.ts" />
-/// <reference path="../StyleEditor/StyleEditor.ts" />
```

Comment:
Looks like your typescript compiler has removed all the .js comments. Was that intentional?
Otherwise, LGTM